clients/mobile/src/donationWorkflow/createUpdateDonation/Donation.tsx

```diff
@@ -91,7 +90,7 @@ const CreateBloodRequest = () => {
             />
           </View>
 
-          <View style={styles.fieldSpacing}>
+          <View style={[styles.fieldSpacing]}>
```

Comment:
Is there any reason to make `style` an array?

Reply:
@EbrahimAli-RU Bhai when debugging the issue, I added that but forgot to remove...